### PR TITLE
add happymodel moblite 6 bnf preset

### PR DESCRIPTION
**Pull request by antony-weber**

### CODE REVIEW

**Comment by sugaarK** [comment]
ok so not so sure how to review some of this... we don't have a default file to suit all the things your changing...

@antony-weber  what's the mission of this preset? to recover basic settings if some one updates?
if so I think using the `defaults` command might  be a good idea at the beginning of the preset..

also if you have a vtx table then you need to include
`#$ INCLUDE_DISCLAIMER: misc/disclaimer/en/vtxtable.txt` 
and it also needs a discussion link
`#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/136`

other wise I think @limonspb needs to have a look as we haven't done any BNF presets yet

**Comment by antony-weber** [comment]
@sugaarK The mission of this preset is to perform some basic setups based on hardware features
The idea is to run it once after flashing with full chip erase and get the whoop ready to fly

**Review by sugaarK — DISMISSED** [review]

File: presets/4.3/bnf/moblite_6.txt [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: BNF
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: HappyModel, Moblite, bnf
+#$ AUTHOR: antony-weber
+#$ DESCRIPTION: Basic settings for a HappyModel Moblite 6 quadcopter
+#$ INCLUDE_DISCLAIMER: misc/disclaimer/en/vtxtable.txt
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/136
+
+defaults
```
Comment (limonspb) [inline presets/4.3/bnf/moblite_6.txt:12]: i think `defaults nasave`. Otherwise, it can reboot, can't it?

File: presets/4.3/bnf/moblite_6.txt [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+#$ OPTION BEGIN (UNCHECKED): Extended VTX power levels
+vtxtable powerlevels 3
+vtxtable powervalues 0 1 2
+vtxtable powerlabels 25 100 200
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): VTX low band
+vtxtable bands 6
+vtxtable band 6 LOW      L CUSTOM  5362 5399 5436 5473 5510 5547 5584 5621
```
Comment (limonspb) [inline presets/4.3/bnf/moblite_6.txt:82]: extra spaces after LOW

**Review by limonspb — CHANGES_REQUESTED** [review]
please make sure you squash commits in one

**Comment by ctzsnooze** [comment]
It would be good to include some really good tune settings.

File: presets/4.3/bnf/moblite_6.txt (hunk 1 above)
Comment (KarateBrot) [reply presets/4.3/bnf/moblite_6.txt:12]: Yeah you're right. Pretty sure it reboots right after `defaults` so it needs to be `defaults nosave`

**Comment by sugaarK** [comment]
@antony-weber  want finish this up?

**Comment by antony-weber** [comment]
@sugaarK yes, i have some ideas how to improve it, and maybe even split vtx settings into different preset and include it to this one

**Comment by sugaarK** [comment]
@antony-weber cool man I just wanted to make sure we hadn't put you off please reach out if you need help

**Comment by ctzsnooze** [comment]
Looks like a useful preset.  Didn't realise the mob6 needed so many hardware customisations.

**Comment by limonspb** [comment]
@antony-weber please address the concerns from above. And more, you included extra file by mistake:
image

File: presets/4.3/bnf/moblite_6.txt [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+#$ OPTION BEGIN (UNCHECKED): SPI FrSky D16
+set rx_spi_protocol = FRSKY_X
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): External CRSF receiver
+serial 1 2048 115200 57600 0 115200
+feature RX_SERIAL
+feature TELEMETRY
+set serialrx_provider = CRSF
+#$ OPTION END
```
Comment (ctzsnooze) [inline presets/4.3/bnf/moblite_6.txt:102]: Maybe put these options inside option_group tags eg
#$ OPTION_GROUP BEGIN: RC Protocol (select one):
...
#$ OPTION_GROUP END

**Comment by antony-weber** [comment]
> It would be good to include some really good tune settings.

Unfortunately my mobile6 is no longer mobile6. Many changes were made to the design that significantly increased its rigidity. My current settings may be too aggressive for a stock moblite6.

**Comment by antony-weber** [comment]
> please make sure you squash commits in one it would probably also make sense to add some description what this preset has. For example, mention that it has no TUNE, mention that user can check for many options etc.

I'll write description next week.I think I will have some free time for this.

**Comment by limonspb** [comment]
closing due to inactivity

**Pull request closed without merging**